content/es/docs/reference/glossary/kube-proxy.md

```diff
@@ -0,0 +1,18 @@
+---
+title: kube-proxy
+id: kube-proxy
+date: 2018-04-12
+full_link: /docs/reference/generated/kube-proxy
+short_description: >
+  `kube-proxy` es un componente de red que se ejecuta en cada nodo del clúster.
+
+aka: 
+tags:
+- fundamental
+- core-object
+---
+ `kube-proxy` es un componente de red que se ejecuta en cada nodo del clúster.
+
+<!--more--> 
+
+`kube-proxy` es responsable del reenvío de peticiones. `kube-proxy` permite el reenvío de flujos TCP y UDP o el reenvío de TCP y UDP mediante el planificador de procesos Round-robin a través de un conjunto de funciones de backend.
```

Comment:
@raelga  Quizá seria conveniente agregar la palabra *paquetes" :

```suggestion
`kube-proxy` es responsable del reenvío de peticiones. `kube-proxy` permite el reenvío de flujo de paquetes TCP y UDP o el reenvío de paquetes TCP y UDP a traves del planificador de procesos Round-robin en conjunto de funciones de backend.
```

Reply:
+1

Reply:
Por otro lado, si puedes @javisabalete, échale un ojo a la revisió que ha hecho @sftim en #15172:

> kube-proxy is a network proxy that runs on each node in your cluster, implementing part of the Kubernetes {{< glossary_tooltip term_id="service">}} concept.
> 
> kube-proxy maintains network rules on nodes. These network rules allow network communication to your Pods from network sessions inside or outside of your cluster.
> 
> kube-proxy uses the operating system packet filtering layer if there is one and it's available. Otherwise, kube-proxy forwards the traffic itself.